website/docs/docs/deploy/webhooks.md

```diff
@@ -26,10 +27,11 @@ You can also check out the free [dbt Fundamentals course](https://courses.getdbt
 :::
 
 ## Prerequisites
-- You have a dbt Cloud account that is on the [Team or Enterprise plan](https://www.getdbt.com/pricing/). 
+- You have a dbt Cloud account that is on the [Team or Enterprise plan](https://www.getdbt.com/pricing/). For `write` access to webhooks: 
+    - Enterprise plan accounts: You must have the [Account Admin](/docs/cloud/manage-access/enterprise-permissions#account-admin), [Admin](/docs/cloud/manage-access/enterprise-permissions#admin), or [Developer](/docs/cloud/manage-access/enterprise-permissions#developer) permission set.
+    - Team plan accounts: You must have a [Developer license](/docs/cloud/manage-access/self-service-permissions) with the [Member](/docs/dbt-cloud-apis/service-tokens#team-plans-using-service-account-tokens) permission set. 
```

Comment:
```suggestion
    - Team plan accounts: You must have a [Member](/docs/dbt-cloud-apis/service-tokens#team-plans-using-service-account-tokens) permission set. 
```

Reply:
I'm suggesting this because I don't really want users to be using their user tokens for webhooks. It's not our best practice here.